.github/workflows/publish-docs.yaml

```diff
@@ -41,6 +41,15 @@ jobs:
           path: docs
           ref: ${{ env.DOCS_BRANCH }}
           token: ${{ secrets.GH_RAD_CI_BOT_PAT }}
+      # Generate resource reference docs
+      - name: Generate resource reference docs
+        run: python docs/.github/scripts/generate_resource_references.py ./radius/hack/bicep-types-radius/generated/ ./docs/docs/content/reference/resources
```

Comment:
This needs to be hooked up to `make generate`. If the files are checked in, then they need to be updated as part of the PR process.

Reply:
Ideally this would run a `make` target.

Reply:
this has been addressed after our offline discussion